ssm/hmm.py

```diff
@@ -408,8 +408,10 @@ def _objective(params, itr):
 
         # Set up the progress bar
         lls = [-_objective(self.params, 0) * T]
-        pbar = trange(num_epochs * M)
-        pbar.set_description("Epoch {} Itr {} LP: {:.1f}".format(0, 0, lls[-1]))
+      #  pbar = trange(num_epochs * M)
```

Comment:
@guyhwilson Looks great. My last request is just to take out these 2 lines here, and then we are g2g